.gitignore

```diff
@@ -182,5 +182,5 @@ versions/run.ver
 
 # jcb checkout and symlinks
 ush/python/jcb
-workflow/jcb
-ci/scripts/jcb
+dev/workflow/jcb
+dev/ci/scripts/jcb
```

Comment:
These can be deleted.  They don't exist at this location.

Reply:
Done.